doc/read-the-docs-site/adr/0003-marconi-indexer-rollbacks.rst

```diff
@@ -0,0 +1,133 @@
+.. _adr3:
+
+ADR 3: Rolling back on-disk data for Marconi indexers
```

Comment:
Do you want to keep it as-is now and modify it later? For this one, removing it may make sense, but I'd understand that we want to keep track of it and just mark it as deprecated in a next PR.

Reply:
Yeah we'd want to keep track of it and mark the state as `Superceeded` when a new one replaces it. For now, we keep it as is.